caikit/runtime/http_server/http_server.py

```diff
@@ -519,7 +542,9 @@ async def _generator() -> pydantic_response:
                     log.error("<RUN51881106E>", err, exc_info=True)
 
                 # If an error occurs, yield an error response and terminate
-                yield ServerSentEvent(data=json.dumps(error_content))
+                yield ServerSentEvent(
+                    data=json.dumps(error_content), event=StreamEventTypes.ERROR.value
```

Comment:
Ok, looks like maybe `str` is intended?

Reply:
Yep in case of error we do dump out the str